notebooks/community/CODEOWNERS

```diff
@@ -17,4 +17,4 @@
 /matching_engine @yinghsienwu
 /neo4j @benofben @htappen
 /sdk/pytorch_lightning_custom_container_training.ipynb @brianchunkang
-/tensorboard @yfang1 @wattli
+/feature_store/* @inardini
```

Comment:
It looks like the /tensorboard line got replaced in this edit

Reply:
Fixed.